src/array.cr

```diff
@@ -1115,12 +1112,18 @@ class Array(T)
     self
   end
 
-  # Equivalent to `Array#select` but makes modification on the current object rather that returning a new one. Returns nil if no changes were made
+  # Modifes `self`, keeping only the elements in the collection for which the passed block returns `true`.
+  # Returns nil if no changes were made.
+  #
+  # Equivalent to `Array#select`
```

Comment:
Maybe "See also" instead of "Equivalent", because they are not equivalent (same goes with reject)